shells/tests/arcs/ts/runtime/multiplexer-integration-test.ts

```diff
@@ -50,7 +49,9 @@ describe('Multiplexer', () => {
       item: consumes s1`;
 
     const thePostsStore = context.stores.find(StoreInfo.isCollectionEntityStore);
-    const postsHandle = await handleForStoreInfo(thePostsStore, {...context, storageService: new DirectStorageEndpointManager()});
+    const postsHandle = await handleForStoreInfo(thePostsStore, {
+      ...context, storageService: new DirectStorageEndpointManager(runtime.driverFactory)
```

Comment:
Can we put the `new DirectStorageEndpointManager(runtime.driverFactory)` incantation somewhere? E.g. `runtime.createDSEManager()`. I think it could be useful for runtime to encapsulate this kind of thing.

Reply:
Thanks for the suggestion! 
Actually, I noticed that I can do better - just used `runtime.storageService` in all (except 2) places.